lighttpd.conf

```diff
@@ -0,0 +1,58 @@
+server.modules += ( "mod_access",
+                    "mod_auth",
+                    "mod_fastcgi",
+                    "mod_alias",
+                    "mod_rewrite",
+                    "mod_openssl",
+                    "mod_accesslog",
+                    "mod_dirlisting")
+
+server.feature-flags = ( "server.network-mptcp" => "enable" )
+server.use-ipv6 = "enable"
+
+$SERVER["socket"] == "[::]:443" {
+    ssl.engine = "enable"
+    ssl.pemfile = "/etc/letsencrypt/live/check.mptcp.dev/fullchain.pem"
+    ssl.privkey = "/etc/letsencrypt/live/check.mptcp.dev/privkey.pem"
+}
+
+$SERVER["socket"] == "0.0.0.0:443" {
+    ssl.engine = "enable"
+    ssl.pemfile = "/etc/letsencrypt/live/check.mptcp.dev/fullchain.pem"
+    ssl.privkey = "/etc/letsencrypt/live/check.mptcp.dev/privkey.pem"
+}
+
+$SERVER["socket"] == "[::]:80" {}
+$SERVER["socket"] == "0.0.0.0:80" {}
+
+$HTTP["url"] =~ "^/\.well-known/acme-challenge/" {
+    server.document-root = "/var/www"
+}
+
+$HTTP["url"] =~ "^/files/" {
+    dir-listing.activate = "enable"
+    server.document-root = "/var/www/"
```

Comment:
should it not be in another dir than the acme-challenge?

also, I don't know if it is linked to the `/` at the end, but if you look at the [files](https://check.mptcp.dev/files/) page, it seems to indicate your files are directories. Any ideas why there is a trailing /?

Reply:
> should it not be in another dir than the acme-challenge?

certbot creates a directory in `/var/www` it unlikely to interfere with anything. the naming scheme of the files and dirs are unique to the domain name the certificate is for. Plus the directory on the host machine is not `/var/www`.

> Any ideas why there is a trailing /?

I believe it is because they do not have a file extension. However the type on the right is correct.

Reply:
> > should it not be in another dir than the acme-challenge?
> 
> certbot creates a directory in `/var/www` it unlikely to interfere with anything. the naming scheme of the files and dirs are unique to the domain name the certificate is for. Plus the directory on the host machine is not `/var/www`.

I meant to say: it looks easy to isolate things, why not doing that then?
- `certbot` would write in a dedicated dir
- the download files would be in another
- the rest in yet another one
 
> > Any ideas why there is a trailing /?
> 
> I believe it is because they do not have a file extension. However the type on the right is correct.

Strange, lighttpd can easily check if it is a file or a directory. Does it change something if you add `.dat` or `.bin` extension?

Reply:
after testing the .bin does not affect the trailing /

Reply:
Is it an issue worth fixing before the merge?

Reply:
If simply removing the extra trailing `/` from all `server.document-root` helps, yes (I guess it is quick to test). If not, fine like that.